packages/teams-js/src/public/interfaces.ts

```diff
@@ -782,11 +819,6 @@ export interface UrlDialogInfo {
    */
   size: DialogSize;
```

Comment:
base dialog has size in it, that means we can delete size from here

Reply:
Deleted in latest commit